_plugins/to_platforms.rb

```diff
@@ -0,0 +1,39 @@
+module Jekyll
+  class ToPlatforms < Jekyll::Generator
+    def generate(site)
```

Comment:
would be nice to document what a Site is right?

Reply:
It's just how Jekyll plugins work. documentation is here https://jekyllrb.com/docs/plugins/ https://jekyllrb.com/docs/plugins/generators/